package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What is this top-level `package-lock.json` file for?

Reply:
thats... actually a good question, I don't know why it got generated, probably by mistakenly running npm install or something like that in the wrong folder? Will delete from branch.